### PR TITLE
WIP: add files for X simulations

### PR DESCRIPTION
**Pull request by ginnocen**

### CODE REVIEW

File: rpythia8/rpythia8.cc [hunk 1]
```diff
@@ -40,6 +41,8 @@ int main(int argc, char **argv)
 
   // pythia, config and init
   Pythia8::Pythia pythia;
+  pythia.particleData.addParticle(9920443, "X(3872)", 3, 0, 0, 3.87196, 0.00012);
```
Comment (preghenella) [inline rpythia8/rpythia8.cc:45]: ciao @ginnocen 
can this particle not be added via configuration files?

**Comment by njacazio** [comment]
Looks good to me! Probably a simple solution would be to use `--seed 0` in the `custom_gen` line of the config file as it should use a time dependent seed but it's fine for me.

**Comment by njacazio** [comment]
Do we want to keep the pythia config in DelphesO2 or shall we put them in O2DPG?

**Comment by ginnocen** [comment]
Ciao @njacazio. I am fine with the solution you think is simpler.

**Pull request closed without merging**